games-server/steamcmd/steamcmd-1.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="This is the command-line version of the Steam client for dedicated servers"
+HOMEPAGE="https://developer.valvesoftware.com/wiki/SteamCMD"
+SRC_URI="https://steamcdn-a.akamaihd.net/client/installer/steamcmd_linux.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+ SteamCMD"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+RDEPEND="app-misc/screen"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}"
+
+pkg_setup() {
+	enewgroup steamcmd
+	enewuser steamcmd -1 -1 /opt/steamcmd steamcmd
+}
+
+src_install() {
+	exeinto /opt/steamcmd
+	doexe steamcmd.sh
+
+	exeinto /opt/steamcmd/linux32
+	doexe linux32/steamcmd linux32/libstdc++.so.6
+
+	newinitd "${FILESDIR}"/steamcmd.initd steamcmd
+	newconfd "${FILESDIR}"/steamcmd.confd steamcmd
+
+	keepdir /opt/steamcmd/.steam/sdk32
+
+	fowners -R steamcmd:steamcmd /opt/steamcmd
+}
+
+pkg_postinst() {
+	elog "Before you can start installing your favourite dedicated server,"
```

Comment:
README.gentoo?

Reply:
Thanks, didn't know, that this exists. This makes the eBuild much cleaner.